### PR TITLE
fixed the bug with routing

### LINKED ISSUE #16: Provide smooth routing facilities .
The routing facilities need to be fixed , on clicking the navbar buttons the page reloads which affects negatively to the user experience . I would like to work on this issue and provide a smooth routing facility .

### PR DESCRIPTION
**Pull request by aayushsrivastava9760**
fixes #16

### CODE REVIEW

**Review by evanjscallan — APPROVED** [review]
LGTM

**Review by DivyanshuParwal — COMMENTED** [review]
Can be closed as this is already being resolved in #14

File: components/Nav.js [hunk 1]
```diff
@@ -72,18 +72,21 @@ export default function Nav({ children }) {
...
-                    isOpen === false
-                      ? navStyles.navlink
-                      : navStyles.navlink + ' ' + navStyles.active
-                  }
-                  onClick={openMenu}
-                >
-                  Listen
-                </a>
-              </Link>
+              {/* <Link href="/https://live.orcasound.net/"> */}
```
Comment (paulcretu) [inline components/Nav.js:75]: I'd just remove code, no point in leaving it commented out

**Comment by paulcretu** [comment]
No longer an issue with #14 merged, closing

**Pull request closed without merging**